Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphService.cs

```diff
@@ -81,6 +81,27 @@ public class MicrosoftGraphService
 
         private string _redirectUri = string.Empty;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MicrosoftGraphService"/> class.
+        /// </summary>
+        public MicrosoftGraphService()
```

Comment:
This should be private.

Reply:
It should be protected otherise MicrosoftGraphCoreService.cs class get affect , I hope protected its fine.

Reply:
what about AuthenticationModel  changes ?

Reply:
That's moving to the MicrosoftGraphService with #2092